src/main/scala/microsites/MicrositesPlugin.scala

```diff
@@ -79,7 +79,14 @@ object MicrositesPlugin extends AutoPlugin {
                             "gray-lighter"    -> "#F4F3F4",
                             "white-color"     -> "#FFFFFF"),
     micrositeGithubOwner := "47deg",
-    micrositeGithubRepo := "sbt-microsites")
+    micrositeGithubRepo := "sbt-microsites",
+    micrositeKazariStyle := "kazari-style-dark",
+    micrositeKazariEvaluatorUrl := "https://scala-evaluator-212.herokuapp.com",
+    micrositeKazariEvaluatorToken := "",
```

Comment:
We just need to generate a new token. The tokens are not to prevent people from accessing it but to identify where they come from and somewhat have an idea of usage. Ultimately we made the decision to gear towards letting client side apps contact the evaluator directly instead of proxying through servers so those tokens will be visible in the client side anyway. Scala Exercise is on of those cases, eventually all exercises will call the evaluator directly instead of through the Scala exercises server.